test/integration/test_seq_copy_int.py

```diff
@@ -148,7 +148,48 @@
      " --batch-size 2 --max-updates 2 --batch-type sentence  --decode-and-evaluate 0"
      " --checkpoint-frequency 2 --optimizer adam --initial-learning-rate 0.01 --lhuc all",
      "--beam-size 2 --beam-prune 1",
-     False, False)]
+     False, False),
```

Comment:
Can you maybe merge some of these tests with the existing ones to avoid adding too many here? I'd suggest using transformer integration tests as these are the default models in Sockeye now.

I would also like to have a system test (one that asserts on ppl and bleu) that uses this new feature (even though its not too interesting to predict length in sequence copy/sort).

Reply:
OK, leaving just 2 tests with the Transformer model. Will submit a revision after fixing scoring test failures again.